ratchet/Makefile

```diff
@@ -8,15 +8,15 @@ schemata/%.json.zst:
 build/%.json: schemata/%.json.zst
 	zstdcat "$^" >"$@"
 
-output/%.cue: ADDRESS=$(shell echo "$*" | tr '_' '/' | cut -d/ -f1-3)
-output/%.cue: build/%.json
-	go run cmd/ratchet/main.go "$^" $(ADDRESS) >"$@"
+testdata/script/unit/datasources/%.txtar.tmp: SOURCE=$(shell echo "$@" | sed -e "s/datasources/resources/" -e "s/.tmp//")
```

Comment:
I /feel/ like this target's recipe could easily be rolled into the recipe of the non-`.tmp` version that uses this file, thereby making the flow clearer. I'm not 100% sure, as I don't have any insight into why these recipes are doing what they're doing.

Reply:
yeah. can we pair to improve this makefile?